packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -65,12 +65,6 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
     setPackSize(newPackSize);
   };
 
-  selectedItem?.availableBatches.forEach(batch => {
-    if (packSizes.every(pack => pack !== batch.packSize)) {
-      packSizes.push(batch.packSize);
-    }
-  });
```

Comment:
A regression, for now - have tried to minimise them and also make small enough changes to PR heh. We have the stock lines in the parent we can use but the summary item doesn't have the stock lines. So thinking to pass an array of pack sizes